tools/ovc/openvino/tools/ovc/moc_frontend/pipeline.py

```
@@ -115,131 +81,195 @@ def add_names_to_tensors(model: InputModel, places: List[Place]):
-
-    if user_shapes:
-        for user_shape in user_shapes:
-            if user_shape.get('shape') is not None:
-                input_model.set_partial_shape(
-                    user_shape['node'], user_shape['shape'])
-            if user_shape.get('data_type') is not None:
-                data_type = user_shape['data_type']
-                log.debug('Set data type: {}'.format(data_type))
-                input_model.set_element_type(user_shape['node'], data_type)
-
-    if freeze_placeholder:
-        for name, value in freeze_placeholder.items():
-            node = None
-            # look for the certain place in user_shapes
-            for node_cur in user_shapes:
-                if node_cur.get('input_name') == name:
-                    node = node_cur
-                    break
-            if node is None:
-                raise Error("Please check correctness of the command-line. "
-                            "Place (operation or tensor) with name {} is not found.".format(name))
-            place = node.get('node')
-
-            if node.get('data_type'):
-                dtype = node['data_type']
-                ov_type = Type(dtype)
+    if getattr(argv, "framework", None) == "pytorch":
+        for idx, input_info in enumerate(argv.input):
+            if getattr(input_info, "name", None):
+                place = input_model.get_place_by_tensor_name(input_info.name)
+                if not input_info.shape and not input_info.type:
+                    # If we received place by name, we need to use it for FE to verify
+                    # that such name exist, otherwise we silently ignore it.
+                    # Using dynamic shape should be safe, because FE will not overwrite
+                    # the shape that was produced after conversion, but merge it, so
+                    # dynamic shape will not change anything.
+                    input_model.set_partial_shape(place, PartialShape.dynamic())
```

Maybe raise error if the place is not found? Or write a warning?

---

We can't verify if place is not found here